docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -2,30 +2,31 @@
 
 You can use the `Tizen.MachineLearning.Inference.SingleShot` class, to load the existing neural network model or your own specific model from the storage. After loading the model, you can invoke it with a single instance of input data. Then, you can get the inference output result.
 
-The main Machine Learning Inference features are:
+The main features of SingleShot include:
 
-- Managing tensor information
+- [Manage tensor information](#manage)
 
   Tensor information is the metadata that contains dimensions and types of tensors.
-  You can [configure the input and the output Tensor Information](#manage) such as name, data type, and dimension.
+  You can configure the input and the output tensor information such as name, data type, and dimension.
 
-- Loading a neural network model and configuring a runtime environment
+- [Load neural network model](#load)
 
-  You can [load the neural network model from storage and configure a runtime environment](#load).
+  You can load the neural network model from storage and configure a runtime environment.
 
-- Invoking the neural network model with input data
+- [Invoke neural network model](#invoke)
 
-  After setting up the SingleShot instance with its required information, you can [invoke the model with the input data and get the inference output result](#invoke).
+  After setting up the SingleShot instance with its required information, you can invoke the model with the input data and get the inference output result.
 
-- Fetching the inference result after invoking
+- [Fetch inference result](#fetch)
```

Comment:
@jaeyun-jung 
Thank you very much for updating all the changes.
The headings are as per the comments.
However, we don't need to change the bulleted list in the heading format.
To maintain consistency can we revert it back?
Please verify.